src/NHSD.BuyingCatalogue.Database/PostDeployment/InsertAdditionalServices.sql

```diff
@@ -187,6 +187,27 @@ BEGIN
 			    (@additionalServiceId, 1, 1, 'F8D06518-1A20-4FBA-B369-AB583F9FA8C0', 1, 'GBP', @now, 599.99);
 	    END;
     END;
+    /***************************************************************************************************************************/
+    SET @solutionId = '100008-001';
+    SET @additionalServiceId = '100008-001-A01'
+
+    IF EXISTS (SELECT * FROM dbo.CatalogueItem WHERE CatalogueItemId = @solutionId)
+    BEGIN
+	    IF NOT EXISTS (SELECT * FROM dbo.CatalogueItem WHERE CatalogueItemId = @additionalServiceId)
+	    BEGIN
+		    INSERT INTO dbo.CatalogueItem(CatalogueItemId, CatalogueItemTypeId, [Name], SupplierId, PublishedStatusId, Created)
+			    VALUES (@additionalServiceId, @additionalServiceItemType, 'Boston Dynamics additional service', '100008', @publishedStatus, @now);
+
+		    INSERT INTO dbo.AdditionalService(CatalogueItemId,Summary,[FullDescription],[LastUpdated],[LastUpdatedBy],[SolutionId])
```

Comment:
A couple of small things: 

1. The square brackets around the column names aren't required here.
2. I also think this would be slightly easier to read with a space after each comma – it would also keep the formatting consistent.